backend/src/main/java/com/mcmasterbaja/analyzer/AnalyzerFactory.java

```diff
@@ -60,6 +60,16 @@ public static Analyzer createAnalyzer(AnalyzerParams params) {
         double b = Double.parseDouble((String) options[1]);
         return new LinearMultiplyAnalyzer(inputFiles, inputColumns, outputFiles, m, b);
 
+      case CONSTANT_ADDER:
+        if (options[3] == "" || options[2] == ""|| options[1] == "" || options[0] == "") {
+            return null;
+        }
+        double a1 = Double.parseDouble((String) options[0]);
+        double b2 = Double.parseDouble((String) options[1]);
+        double c1 = Double.parseDouble((String) options[2]);
+        double d1 = Double.parseDouble((String) options[3]);
+        return new ConstantAdderAnalyzer(inputFiles, inputColumns, outputFiles, a1, b2,c1,d1);  
```

Comment:
Why is this named a1, b2, c1, d1?

Reply:
Assuming this is because of variable names defined multiple times, this is fine I should fix this later